Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR_test.py

```diff
@@ -1580,3 +1580,40 @@ def test_get_xsoar_close_reasons(mocker):
     }
     mocker.patch.object(demisto, 'internalHttpRequest', return_value=mock_response)
     assert get_xsoar_close_reasons() == list(XSOAR_RESOLVED_STATUS_TO_XDR.keys()) + ['CustomReason1', 'CustomReason 2', 'Foo']
+
+
+def test_get_modified_remote_data_xdr_delay(mocker):
+    """
```

Comment:
I would like us to:
1. Assert on the values set to the API in the HTTP request.
2. Add all use cases relevant for this change.